core/geometry/src/test/topology/RegionOps.test.ts

```diff
@@ -1162,3 +1257,163 @@ describe("RegionOps2", () => {
     expect(ck.getNumErrors()).equals(0);
   });
 });
+
+describe("RegionOps.constructPolygonWireXYOffset", () => {
```

Comment:
These tests only allow for visual verification of the output. Are there ways you can verify the output in code? Typically, we want to automate this checking where feasible so that when the test runs, it can detect some regressions. E.g.:
* offset has arcs at expected joints per options
* offset has sharp corners at expected joints per options
* offset has chamfer linestrings at expected joints per options
* open/closed input results in open/closed offset as expected per wrap value
* sampled points on naïve offset segments have expected offset distance from input
* offset is on expected side of the input curve

Some of these may already be verified by existing tests.